app/upgrades.go

```diff
@@ -40,6 +40,8 @@ var handlers = map[string]appUpgrade{
 	"ochre-rc1": { // upgrade for 1.13.0-rc1
 		Added: []string{group.ModuleName, rewardtypes.ModuleName, icacontrollertypes.StoreKey, icahosttypes.StoreKey},
 		Handler: func(ctx sdk.Context, app *App, plan upgradetypes.Plan) (module.VersionMap, error) {
+			params := app.MsgFeesKeeper.GetParams(ctx)
+			app.MsgFeesKeeper.SetParams(ctx, params)
```

Comment:
`ochre-rc1` has already run on testnet, so this ends up doing nothing.

We can't just change `ochre-rc1` to `ochre` for the mainnet upgrade either. Doing so would make `provenanced` not work on testnet anymore because it'd be missing the upgrade handler.

So... This either needs to be added to a new `ochre-rc2` upgrade handler (if we plan on doing another rc upgrade in testnet) or else we need to copy the existing `ochre-rc1` handler to a new `ochre` one, and add these lines there.

If we're going to skip an `-rc2` upgrade, we should probably have someone put in a gov proposal on testnet to fix the params on there.

Reply:
i agree with @SpicyLemon we need to take this out of `ochre-rc1` and add it to `ochre` (which will be added when we are ready to release to mainnet imo. We should take out those two lines ^^

Reply:
Agree, I was thinking we would change `ochre-rc1` to `ochre`...but that is wrong.

@SpicyLemon we already did a gov proposal for the fix before this PR.  So, we are good on that front.